File: 2025-Tecnicas-Programacion/Clase-14/Python/clase-2-3.py
"""
evaluar si se puede dividir por 0
"""
n1=int(input("Ingresar un numero "))
n2=int(input("Ingrese otro numero "))
#evaluo si el n2 es un 0
if(n2==0):
    print("No se puede dividir por 0")
else:
    print(f" la division es: {n1/n2}")
    
#transformar a operador ternario
print(f"No se puede dividir por 0" if(n2==0) else "La division es:",n1/n2)

File: 2025-Relaciones-Laborales/Clase-2/html/clase2-3.js
//  condicionales
numero1 = parseInt(prompt("Ingrese un numero"));
numero2 = parseInt(prompt("Ingrese otro numero"));
//agregamos condicion divisor == 0
if (numero2 == 0) {
    console.error("No se puede dividir por 0")
} else {
    alert("La division es: " + numero1 / numero2);
}
